core/corehttp/gateway_handler.go

```diff
@@ -387,14 +383,14 @@ func (s *sizeSeeker) Seek(offset int64, whence int) (int64, error) {
 	return s.sizeReadSeeker.Seek(offset, whence)
 }
 
-func (i *gatewayHandler) serveFile(w http.ResponseWriter, req *http.Request, name string, modtime time.Time, content io.ReadSeeker) {
+func (i *gatewayHandler) serveFile(w http.ResponseWriter, req *http.Request, modtime time.Time, content io.ReadSeeker) {
```

Comment:
I would keep this parameter but ignore it, to make it easier to change the logic later down the road, just add a comment that the name is ignored _for now_.

Reply:
SGTM.